sic_framework/devices/nao.py

```diff
@@ -67,7 +67,7 @@ def sic_install(self):
                     pip install social-interaction-cloud --no-deps;
                     pip install Pillow PyTurboJPEG numpy redis six;
                                         
-                    if [ -d /data/home/nao/.venv_sic/lib/python2.7/site-packages/sic_framework ]; then
+                    if [ pip list | grep -w 'social-interaction-cloud' > /dev/null 2>&1 ]; then
```

Comment:
The bracket only works for test operators. So the bracket here breaks the pipeline and an error message like this will occur: `bash: line 16: [: missing `]'`

Reply:
The 7270f1c0c4b024e98aae5da948692e1a554998d9 has fixed this